packages/create-atomic/scripts/preparePackageJsonTemplate.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+# Copy ./templates/package.json to ./templates/package.json.hbs, and:
+#   * Replace name by "{{project}}"
+#   * Add "postinstall": "npm run setup-lambda && npm run setup-cleanup" to the scripts
+jq '.name="{{project}}" | .scripts.postinstall= "npm run setup-lambda && npm run setup-cleanup"' ./templates/package.json > ./templates/package.json.hbs
```

Comment:
You could use append or modify I think
https://plopjs.com/documentation/#append

Example here https://github.com/plopjs/plop/blob/master/example/typescript/plopfile.ts#L118

Or leave it as is for now and I'll rework on the plopfile soon anyways

Reply:
I could have used JavaScript indeed, we do it at other places to tweak package.json.

The issue is the formating, (tabs, spaces, how many...), there's no good way to preserve it easily with Node.
`jq` respect the formating ootb, which made it the simplest solution IMO

Reply:
Changed for a JS script to ensure win-compat